src/components/Orders/OrderList.jsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from "react";
+import { getAllOrders, deleteOrder } from "../../api/orderService";
+import OrderForm from "./OrderForm";
+
+const OrderList = () => {
+	const [orders, setOrders] = useState([]);
+
+	useEffect(() => {
+		loadOrders();
+	}, []);
+
+	const loadOrders = async () => {
+		const orders = await getAllOrders();
+		setOrders(orders);
+	};
+
+	const handleDelete = async (id) => {
+		await deleteOrder(id);
+		loadOrders();
+	};
+
+	return (
+		<div className="container mx-auto p-4">
+			<h1 className="text-3xl font-bold mb-4">Orders</h1>
+			<OrderForm onOrderAdded={loadOrders} />
```

Comment:
we should only keep order list related code in this file, orderForm component can be added on page/Orders.js